gradle/libs.versions.toml

```diff
@@ -14,55 +14,58 @@
 ## limitations under the License.
 ##
 [versions]
-accessibilityTestFramework = "4.0.0"
-activityCompose = "1.8.1"
-androidGradlePlugin = "8.1.2"
-benchmark = "1.1.0"
+accessibilityTestFramework = "4.1.0"
+activityCompose = "1.8.2"
+androidGradlePlugin = "8.2.1"
+benchmark = "1.2.2"
 # @keep
 compileSdk = "34"
-composeLatest = "1.6.0-beta03"
+compose-compiler = "1.5.7"
 composeBom = "2023.10.01"
-compose-compiler = "1.5.3"
+composeLatest = "1.6.0-beta03"
 constraintLayoutCompose = "1.0.1"
 coreTesting = "2.2.0"
-coroutines = "1.6.4"
-espresso = "3.4.0"
+coroutines = "1.7.3"
+espresso = "3.5.1"
 glide = "1.0.0-beta01"
 gradle = "7.2.0"
+gradle-versions = "0.50.0"
 gson = "2.9.0"
-guava = "31.1-android"
-hilt = "2.48"
-hiltNavigationCompose = "1.0.0"
+guava = "33.0.0-jre"
+hilt = "2.50"
+hiltNavigationCompose = "1.1.0"
 junit = "4.13.2"
-kotlin = "1.9.10"
+kotlin = "1.9.21"
+ksp = "1.9.21-1.0.15"
 ktlint = "0.40.0"
 ktx = "1.7.0"
-lifecycle = "2.6.0-alpha04"
+lifecycle = "2.6.2"
 material = "1.8.0-rc01"
-material3 = "1.2.0-alpha11"
+material3 = "1.2.0-beta01"
 # @keep
-minSdk = "23"
-monitor = "1.6.0"
-navigation = "2.5.3"
-okhttpLogging = "4.10.0"
-pagingCompose = "1.0.0-alpha19"
-profileInstaller = "1.2.0"
+minSdk = "28"
```

Comment:
This is a pretty big jump; intentional?

Reply:
I think I bumped this while debugging, but it doesn't have to be 28. Will revert.